Test.cs

```diff
@@ -4,6 +4,8 @@ namespace FactorialCalculator
 {
     public class Factorial
     {
+        private branchName = "Dev"
+        private int id = 10;
```

Comment:
The `branchName` and `id` fields should have their types explicitly defined. Furthermore, `branchName` should be a constant if its value does not change. Please use PascalCase for public constants and camelCase for private fields.